packages/blob/src/client.browser.test.ts

```diff
@@ -20,6 +20,7 @@ jest.mock('undici', () => ({
       json: () =>
         Promise.resolve({
           url: `https://storeId.public.blob.vercel-storage.com/superfoo.txt`,
+          downloadUrl: `https://storeId.public.blob.vercel-storage.com/superfoo.txt?download=1`,
```

Comment:
I wonder why the refactor requires to update the tests? I thought the feature would have required the test to be updated already

Reply:
because it resolved to `downloadUrl: undefined`